db/seeds/gear.seeds.rb

```diff
@@ -1,103 +1,101 @@
+    { name: 'Pocket Knife', harm: 1, tags: 'hand, useful, small', playbooks: [@mundane] },
+    { name: 'Protective Wear', armor: 1, tags: '', playbooks: [@wronged] },
+    { name: 'Razor Whip', harm: 3, tags: 'hand, area, messy, holy', playbooks: [@divine] },
+    { name: 'Ritual Knife', harm: 1, tags: 'hand', playbooks: [@spell_slinger] },
+    { name: 'Sawn-off Shotgun', harm: 3, tags: 'hand/close, messy, loud, reload', description: 'signature', playbooks: [@wronged] },
+    { name: 'Shotgun', harm: 3, tags: 'close, messy, loud', description: 'signature', playbooks: [@expert, @flake, @wronged] },
+    { name: 'Shotgun', harm: 3, tags: 'close, messy', description: 'normal, modern', playbooks: [@crooked, @initiate, @monstrous, @professional, @spooky] },
+    { name: 'Silver Knife', harm: 1, tags: 'hand, silver', playbooks: [@initiate], description: 'old-fashioned' },
+    { name: 'Silver Sword', harm: 2, tags: 'hand, messy, silver', playbooks: [@expert] },
+    { name: 'Silver Trident', harm: 3, tags: 'hand, silver, holy', playbooks: [@divine] },
+    { name: 'Skateboard', tags: '', playbooks: [@mundane], description: 'mundane_transport' },
+    { name: 'Sledgehammer', harm: 3, tags: 'hand, messy', playbooks: [@mundane] },
+    { name: 'Small Handgun', harm: 2, tags: 'close, loud, reload', playbooks: [@mundane] },
+    { name: 'Sniper Rifle', harm: 4, tags: 'far', description: 'serious', playbooks: [@professional] },
+    { name: 'Sniper Rifle', harm: 3, tags: 'far', description: 'serious, modern', playbooks: [@initiate] },
+    { name: 'Spear', harm: 2, tags: 'hand/close', playbooks: [@initiate], description: 'old-fashioned' },
+    { name: 'Specialist Weapon (Other Foe)', harm: 1, tags: "Keeper's Discretion", description: 'signature', playbooks: [@wronged] },
+    { name: 'Specialist Weapons (Chosen Foe)', harm: 4, tags: "Keeper's Discretion", description: 'signature', playbooks: [@wronged] },
+    { name: 'Submachine Gun', harm: 2, tags: 'close, reload, area', playbooks: [@crooked] },
+    { name: 'Submachine Gun', harm: 3, tags: 'close, area, loud, reload', description: 'serious', playbooks: [@professional] },
+    { name: 'Sword', harm: 2, tags: 'hand, messy', playbooks: [@initiate, @monstrous], description: 'old-fashioned' },
+    { name: 'Throwing Knives', harm: 1, tags: 'close, many', playbooks: [@flake] },
+    { name: 'Thunder Hammer', harm: 3, tags: 'hand, stun, holy', playbooks: [@divine] },
+    { name: 'Truck', tags: '', description: 'professional_transport', playbooks: [@professional] },
+    { name: 'Van', tags: '', description: 'mundane_transport, professional_transport', playbooks: [@mundane, @professional] },
+    { name: "Watchman's Flashlight", harm: 1, tags: 'hand', playbooks: [@flake] },
+    { name: 'Weighted Gloves/Brass Knuckles', harm: 1, tags: 'hand', playbooks: [@flake] }
+  ].each do |gear|
+    tag_list = gear.delete(:tags)
+    playbooks = gear.delete(:playbooks)
+    g = Gear.create!(gear)
```

Comment:
What if the gear already exists? We run our seed files every deploy. I believe this would constantly recreate gear in our database.

Reply:
Corrected to `find_or_initialize_by` since the save is called later in the code.  Resolved in commit 3501c33.